### PR TITLE
Added templates for all repeating and non-repeating datatypes.

### PR DESCRIPTION
**Pull request by yash-ni**

### CODE REVIEW

File: src/grpc_client.cc [hunk 1]
```diff
@@ -609,14 +609,14 @@ LIBRARY_EXPORT int32_t ClientCompleteReadFromStream(grpc_labview::gRPCid* callId
     *success = reader->_readFuture.get();
     if (*success)
     {
-        try
+        /*try
```
Comment (CPattar-NI) [inline src/grpc_client.cc]: Use "if (!grpc_labview::g_use_hardcoded_parse)" instead of commenting it out.

Comment (CPattar-NI) [reply src/grpc_client.cc]: Same goes for all the other calls.

File: src/lv_message.cc [hunk 2]
```diff
@@ -116,6 +116,12 @@ namespace grpc_labview
                 {
                     auto fieldInfo = (*fieldIt).second;
                     LVMessageMetadataType dataType = fieldInfo->type;
+                    if (_LVClusterHandle == nullptr) {
```
Comment (CPattar-NI) [inline src/lv_message.cc:119]: You don't need this anymore right?

Comment (yash-ni) [reply src/lv_message.cc:119]: No it is still needed to enable labview server

**Pull request merged**